db/migrate/20170801160245_create_tags.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTags < ActiveRecord::Migration[5.0]
```

Comment:
Would like to have seen you start with a test that would have pushed you to creating tags.